tests/indexes/models.py

```diff
@@ -38,18 +38,17 @@ class Article(models.Model):
     )
 
     class Meta:
-        index_together = [
-            ["headline", "pub_date"],
-        ]
+        indexes = [models.Index(fields=["headline", "pub_date"])]
 
 
 # Model for index_together being used only with single list
 class IndexTogetherSingleList(models.Model):
     headline = models.CharField(max_length=100)
     pub_date = models.DateTimeField()
 
-    class Meta:
-        index_together = ["headline", "pub_date"]
+    # Added on-the-fly by the test method.
+    # class Meta:
+    #     index_together = ["headline", "pub_date"]
```

Comment:
This model is used only by a single test method. I'd made it inline in advance:
```python
    def test_index_together_single_list(self):
        # Test for using index_together with a single list (#22172)
        class IndexTogetherSingleList(Model):
            headline = CharField(max_length=100)
            pub_date = DateTimeField()
            
            class Meta:
                index_together = ["headline", "pub_date"]
            
        index_sql = connection.schema_editor()._model_indexes_sql(
            IndexTogetherSingleList
        )   
        self.assertEqual(len(index_sql), 1)
```

Reply:
For both test model classes that weren't inlined, I already tried it but some unit tests then fail because of them:
```
proxy_model_inheritance.tests.ProxyModelInheritanceTests.test_table_exists
test_runner.test_debug_sql.TestDebugSQL.test_output_normal
test_runner.test_debug_sql.TestDebugSQL.test_output_verbose
test_runner.test_debug_sql.TestDebugSQL.test_setupclass_exception
```
https://djangoci.com/job/pull-requests-bionic/database=sqlite3,label=bionic-pr,python=python3.8/16216/
or just `proxy_model_inheritance.tests.ProxyModelInheritanceTests.test_table_exists` in some cases
https://djangoci.com/job/pr-mariadb/database=mysql,label=mariadb,python=python3.10/16210/
due to model migration it seems.

The common thing about these test cases are calling the `migrate` command (with `run_syncdb=True`)
- `tests.proxy_model_inheritance.tests.ProxyModelInheritanceTests.test_table_exists` does directly
- `test_runner.test_debug_sql.TestDebugSQL` call it through `_test_output`, which sets up databases and end up in `django.db.backends.base.creation.BaseDatabaseCreation.create_test_db`